FactorialTask/Properties/AssemblyInfo.cs

```diff
@@ -4,11 +4,11 @@
 // General Information about an assembly is controlled through the following 
 // set of attributes. Change these attribute values to modify the information
 // associated with an assembly.
-[assembly: AssemblyTitle("FactorialTask")]
+[assembly: AssemblyTitle("FactorialSubtask")]
```

Comment:
I think the assembly name can stay at `FactorialTask` since that is also the name of the used namespace